plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_aws_launcher.py

```diff
@@ -21,43 +22,47 @@
     ray_rsync_up,
     ray_tmp_dir,
     ray_up,
+    rsync_up,
 )
 
 log = logging.getLogger(__name__)
 
 
 class RayAWSLauncher(Launcher):
     def __init__(
-            self,
-            ray_init_cfg: DictConfig,
-            ray_remote_cfg: DictConfig,
-            ray_cluster_cfg: DictConfig,
-            stop_cluster: bool,
-            code_dir: str,
+        self,
+        ray_init_cfg: DictConfig,
+        ray_remote_cfg: DictConfig,
+        ray_cluster_cfg: DictConfig,
+        stop_cluster: bool,
+        download_log: bool,
+        code_sync_cfg: DictConfig,
```

Comment:
there two can maybe become:
```python
sync_up : RsyncConfig,
sync_down : RsyncConfig,
```
Both can potentially share the same config type, but just have different defaults.
The behavior of what each one is relative to should be documented.